test/parallel/test-async-hooks-promise-triggerid.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const async_hooks = require('async_hooks');
+
+const promiseAsyncIds = [];
+
+async_hooks.createHook({
+  init: common.mustCallAtLeast((id, type, triggerId) => {
+    if (type === 'PROMISE') {
+      assert.strictEqual(promiseAsyncIds[promiseAsyncIds.length - 1] || 1,
```

Comment:
why test that the triggerId propagates correctly? Maybe you were thinking of testing `async_hooks.triggerId()`?